tests/project-list.spec.ts

```diff
@@ -0,0 +1,66 @@
+// modules
+import { expect, test } from "@playwright/test";
+import { capitalize } from "lodash";
+
+// files
+import projectsMockedResponseBody from "./fixtures/projects.json";
```

Comment:
Small tiny 🌵 but it's a long name. `projectsMock` or `mockProjectsRes` or whatever. No need to change.